pom.xml

```diff
@@ -1844,51 +1844,14 @@
             <version>${pitest.plugin.version}</version>
             <configuration>
               <targetClasses>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationOnSameLineCheck
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationUseStyleCheck
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.MissingDeprecatedCheck
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.MissingOverrideCheck
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.PackageAnnotationCheck
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.SuppressWarningsCheck
-                </param>
+                <param>com.puppycrawl.tools.checkstyle.checks.annotation.*</param>
               </targetClasses>
               <targetTests>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheckTest
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationOnSameLineCheckTest
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationUseStyleCheckTest
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.MissingDeprecatedCheckTest
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.MissingOverrideCheckTest
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.PackageAnnotationCheckTest
-                </param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.annotation.SuppressWarningsCheckTest
-                </param>
+                <param>com.puppycrawl.tools.checkstyle.checks.annotation.*</param>
               </targetTests>
+              <excludedTestClasses>
+                <param>*.Input*</param>
```

Comment:
Hmm, a lot of wildcards , do we still have inconveniences in names of input files ?

Reply:
> a lot of wildcards

the `.` before `Input` ensures Input is the start of the file name.

> do we still have inconveniences in names of input files ?

This only pertains to the test resource directory and java files.
I did a search and the only other things showing up was `package-info`, `AAClass`, etc. 26 files in all out of 960 don't start with `Input`.

This will atleast help remove the majority of the input files. If you know a way to specify the directory, instead of package/class names, then that would be the better option but I am not seeing an exclude directory option.

Reply:
> the . before Input ensures Input is the start of the file name.

ok ,this is not a regexp.
http://pitest.org/quickstart/maven/#excludedtestclasses
https://en.wikipedia.org/wiki/Glob_(programming)